crates/router/src/connector/bluesnap/transformers.rs

```diff
@@ -545,7 +550,7 @@ impl<F, T>
     ) -> Result<Self, Self::Error> {
         Ok(Self {
             response: Ok(types::PaymentsResponseData::ConnectorCustomerResponse {
-                connector_customer_id: item.response.vaulted_shopper_id.to_string(),
+                connector_customer_id: item.response.vaulted_shopper_id.peek().to_string(),
```

Comment:
Hey, its only my personal preference, but can we consider not using `peek()`, `.expose()` is much more concrete and consumes self in case of transformation or consumption